sdk/python/kfp/v2/components/pipeline_task.py

```diff
@@ -500,6 +500,22 @@ def set_display_name(self, name: str) -> 'PipelineTask':
         self.task_spec.display_name = name
         return self
 
+    def set_env_variable(self, name: str, value: str) -> 'PipelineTask':
```

Comment:
There's also a disable test in compiler_cli_test, can you reenable that:
https://github.com/kubeflow/pipelines/blob/cce453aa309b33976b397b0a3935da06756118bb/sdk/python/kfp/v2/compiler_cli_tests/compiler_cli_tests.py#L175-L177